README.md

```diff
@@ -1,161 +1,187 @@
-# HoloSqape
+# Qt Holochain Containers
 
 [![Project](https://img.shields.io/badge/project-holochain-blue.svg?style=flat-square)](http://holochain.org/)
 [![PM](https://img.shields.io/badge/pm-waffle-blue.svg?style=flat-square)](https://waffle.io/holochain/org)
 [![Chat](https://img.shields.io/badge/chat-chat%2eholochain%2enet-blue.svg?style=flat-square)](https://chat.holochain.net)
 
-A multiplatform Holochain app container that runs in the background and is accessible via a system tray icon based on Qt, QML and the C bindings of [holochain-rust](https://github.com/holochain/holochain-rust).
+Multiplatform Holochain containers for the installation and execution of hApps based on Qt, QML and the C bindings of [holochain-rust](https://github.com/holochain/holochain-rust).
 
-![](holosqape/images/screenshots/system-tray.png)
-![](holosqape/images/screenshots/mainwindow-with-dock.png)
-
-## Dev Dependencies
-
-* [Rust](https://www.rust-lang.org/en-US/install.html)
-* [Qt](http://doc.qt.io/qt-5/index.html) (On macOS, `brew install qt` should be fine.)
-* [cmake](https://cmake.org/install)
+This repository contains two different containers, both based on the same Qt <> C bindings of `holochain-rust`. There are several sub directories in this repo containing separate sub projects.
```

Comment:
The thing is that currently is not completely correct to say that those are two different containers. They share the same configuration. So if you install an app in Holosqape, you can instantiate it in holoconsole as well, and vice versa.. Holoconsole is basically the same thing, but without GUI.

Reply:
Ah, I see